docs/.vuepress/theme/components/SchemaStructuredData.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <div>
+    <script
+      v-if="meta_structuredData"
+      type="application/ld+json"
+      v-html="meta_structuredData"
+    ></script>
+  </div>
+</template>
+
+<script>
+import * as dayjs from 'dayjs';
+import utc from 'dayjs/plugin/utc';
+
+const timezone = require('dayjs/plugin/timezone');
```

Comment:
Is there any chance of using `import` here? It looks a bit odd to see ESM and CommonJS mixed together 😄